CMakeLists.txt

```diff
@@ -1,23 +1,45 @@
-project(MVDTool)
 cmake_minimum_required(VERSION 3.0)
 
-include(CTest)
-
-set(MVDTOOL_VERSION_MAJOR 1)
-set(MVDTOOL_VERSION_MINOR 5)
-set(MVDTOOL_VERSION_PATCH 2)
+# get version from Python package
+find_package(PythonInterp REQUIRED)
```

Comment:
check out this instead:
https://github.com/BlueBrain/nmodl/blob/master/cmake/FindPythonModule.cmake

Reply:
Looks good, but isn't that for module dependencies? We're trying to get just the version out of `setup.py`, without requiring any python modules to be installed, really.